src/main/java/net/sf/jabref/gui/preftabs/NetworkTab.java

```diff
@@ -48,31 +58,60 @@ public NetworkTab(JabRefPreferences prefs) {
         defProxyPort = new JTextField();
         defProxyPort.setEnabled(false);
 
+        useProxyAuthentication = new JCheckBox(Localization.lang("Proxy requires authentication"));
+        useProxyAuthentication.setEnabled(false);
+
+        defProxyUsername = new JTextField();
+        defProxyUsername.setEnabled(false);
+        defProxyPassword = new JPasswordField();
+        defProxyPassword.setEnabled(false);
+        lblPasswordWarning = new JLabel(Localization.lang("Attention: Password is stored in plain text!"));
+        lblPasswordWarning.setEnabled(false);
```

Comment:
Indeed, but it is in general a good practice to disable irrelevant labels to not disturb/irritate the user. I can change it to be kept enabled as long as the password is not empty. That is IMHO the best compromise. Do you agree?